qemu/tests/cfg/win_virtio_driver_update_test.cfg

```diff
@@ -177,3 +177,17 @@
             python_bin = python2.7
             file_sender = guest
             target_process = ${python_bin}.exe
+        - with_vioinput:
+            no Host_RHEL.m6 Host_RHEL.m7.u0 Host_RHEL.m7.u1 Host_RHEL.m7.u2 Host_RHEL.m7.u3
+            no Win2008..sp2
+            only before_bg_test
+            driver_name = "vioinput"
+            device_name = "VirtIO Input Driver"
+            device_hwid = '"PCI\VEN_1AF4&DEV_1052"'
+            inputs = input1
+            input_dev_bus_type_input1 = virtio
+            variants:
+                - device_keyboard:
+                    input_dev_type_input1 = keyboard
+                    run_bgstress = "vioinput_keyboard"
+                    key_table_file = key_to_keycode_win.json
```

Comment:
Please update all the configurations according to the changes of https://github.com/autotest/tp-qemu/pull/1853

Reply:
updated, thanks~